File: Easy/3014_Minimum_Number_of_Pushes_to_Type_Word_I/Solution.py
from typing import List, Optional


class Solution:
    def minimumPushes(self, word: str) -> int:
        ans = 0

        for i in range(len(word)):
            ans += i // 8 + 1

        return ans
        

if __name__ == "__main__":
    solution = Solution()

    # test cases 1
    word1 = "abcde"
    print(solution.minimumPushes(word1))

    # test cases 2
    word2 = "xycdefghij"
    print(solution.minimumPushes(word2))
    

File: Easy/3014_Minimum_Number_of_Pushes_to_Type_Word_I/Solution.java



public class Solution {
    public int minimumPushes(String word) {
        int ans = 0;

        for (int i = 0; i < word.length(); i++) {
            ans += i / 8 + 1;
        }

        return ans;
    }

    public static void main(String[] args) {
        Solution solution = new Solution();

        // test cases 1
        String word1 = "abcde";
        System.out.println(solution.minimumPushes(word1));

        // test cases 2
        String word2 = "xycdefghij";
        System.out.println(solution.minimumPushes(word2));
        
    }
}